src/components/DonateButton.jsx

```diff
@@ -1082,23 +1082,42 @@ export default class Root extends React.PureComponent {
 
   afterSuccessfulDonate() {
     const { donateToDefaultDac } = this.state;
+    const { customThanksMessage } = this.props.model;
+
+    const el = document.createElement('div');
+    el.innerHTML = customThanksMessage;
 
     if (!this.props.currentUser || this.props.currentUser.name) {
       // known user
       if (donateToDefaultDac) {
-        React.swal({
-          title: 'Thank you!',
-          text: 'Would you like to support Giveth as well?',
-          icon: 'success',
-          buttons: ['No Thanks', 'Support Giveth'],
-        }).then(result => {
-          if (result) {
-            this.defaultDacDonateButton.current.openDialog();
-          }
-        });
+        if (customThanksMessage !== '') {
+          // Custom Thanks for Commons Stack
+          React.swal({
+            title: 'Thank you!',
+            content: el,
+            icon: 'success',
+            buttons: ['Ok!'],
+          }).then(result => {
+            if (result) {
+              this.defaultDacDonateButton.current.openDialog();
```

Comment:
It's different than what was mentioned here: https://github.com/Giveth/giveth-dapp/issues/1413#issuecomment-714582026